documentation/TechnicalDesignDocument.tex

```diff
@@ -45,6 +45,25 @@
     %------------------------------------------------------------------------------------------------------------------------------------------------------- Requirements
     \section{Requirements}
     Functional and Non-functional requirements.
+    \subsection{Functional Requirements:}
+    \begin{itemize}
+        \item The program should support loading different Marama shapes using the Marama transpiler.
+        \item The program should let users add Marama objects to a world.
+        \item The program should let users see the Marama objects from any angle.
+        \item The program should allow users to attach Maramas to one another by their joints.
+        \item The program should check if attaching Maramas to one another is possible.
+        \item The program should support algebraic data structure Marama shapes.
+        \item The program should support polymorphic Marama shapes.
+        \item A game program should exist where the user can use the editor with predefined maps and limited Marama options.
```

Comment:
For this line and all the lines above (up until l. 51) you use the word marama a lot. Marama is just the language.

I think maramafication is the word you are looking for.

Reply:
A 'Marama' is the object that is created after the process of maramafication gets done on a programming construct. Visupol is the language.

Reply:
I disagree. A maramafication is the object that is returned by the transpiler. Marama is a program, not an object, see Chide his definition in the devlog.
If you read the 'Summary of assignment' in the devlog or the goal of the transpiler then this should become clear to you.